pyecore/resources/json.py

```diff
@@ -4,6 +4,8 @@
 from enum import unique, Enum
 from functools import lru_cache
 import json
+
+from pyecore.ecore import EReference
```

Comment:
Could you use the `ecore` module imported below or use a relative import instead? I know they are discussions about this, but I would prefer in order to be consistent with the rest of the code